lib/events.js

```diff
@@ -1032,8 +1033,22 @@ async function _validateGraph({eventHashes, eventMap, ledgerNode}) {
               [directAncestor] = await api.getEvents(
                 {eventHash: treeHash, ledgerNode});
             }
-            const {meta: {continuity2017: {creator, type}}} = directAncestor;
-            if(!(type === 'm' && creator === requiredCreator)) {
+            const {meta: {continuity2017: {creator, type, generation}}} =
+              directAncestor;
+            if(type !== 'm') {
+              throw new BedrockError(
+                'Peers must base regular events on a merge event.',
+                'DataError', {
+                  ancestorRecord,
+                  directAncestor,
+                  eventRecord,
+                  httpStatusCode: 400,
+                  public: true,
+                });
+            }
+            // a creator mismatch is acceptable if the event is based on the
+            // generation 0 merge event (genesis merge event)
```

Comment:
The comment should probably read the opposite since the condition is checking for an error ... so more like "creators must match after the genesis merge event (after generation 0)"